llvm/lib/MCCAS/MCCASObjectV1.cpp

```diff
@@ -3172,25 +3165,151 @@ static AbbrevEntryReader getAbbrevEntryReader(ArrayRef<StringRef> AbbrevEntries,
   return AbbrevEntryReader(AbbrevData);
 }
 
-Error DIEVisitor::visitDIERef(BinaryStreamReader &DataReader,
-                              unsigned AbbrevIdx, StringRef DIEData,
-                              ArrayRef<DIEDataRef> &DIEChildrenStack) {
+static std::optional<uint8_t> getNonULEBFormSize(dwarf::Form Form,
+                                                 dwarf::FormParams FP) {
+  switch (Form) {
+  case dwarf::DW_FORM_addr:
+  case dwarf::DW_FORM_ref_addr: {
+    auto FormSize =
+        (Form == dwarf::DW_FORM_addr) ? FP.AddrSize : FP.getRefAddrByteSize();
+    return FormSize;
+  }
```

Comment:
This line made me stop to figure out if there is something special going on here, but there is no reason to combine these `case`s?

```
  case dwarf::DW_FORM_addr:
    return FP.AddrSize
  case dwarf::DW_FORM_ref_addr:
    return FP.getRefAddrByteSize();
```

Reply:
Yeah you are right, oversight on my part, sorry